com.unity.multiplayer.transport.utp/Runtime/UTPTransport.cs

```diff
@@ -260,14 +260,14 @@ public void SendToClient(NativeArray<byte> packet, ulong clientId, int index)
         }
     }
 
-    public override unsafe void Send(ulong clientId, ArraySegment<byte> data, NetworkChannel networkChannel)
+    public override unsafe void Send(ulong clientId, ArraySegment<byte> data, NetworkDelivery networkDelivery)
     {
         var pipelineIndex = 0;
 
         GetUTPConnectionDetails(clientId, out uint peerId);
 
         var writer = new DataStreamWriter(data.Count + 1 + 4, Allocator.Temp);
-        writer.WriteByte((byte)networkChannel);
+        writer.WriteByte((byte)networkDelivery);
```

Comment:
I'm assuming actually all these writes can be deleted, because this is probably so that the receiver can route the packet to the right channel, and if we don't care about channels this can be deleted.

Reply:
oh yeah, that'd be great!

Reply:
That is cirrect we can get rid of them.